src/client/sandbox/native-methods.js

```diff
@@ -138,6 +140,8 @@ class NativeMethods {
         // Event
         document.addEventListener    = this.documentAddEventListener;
         document.removeEventListener = this.documentRemoveEventListener;
+
+        this.isDocumentMethodsSaved  = false;
```

Comment:
I don't get it. Why we need this flag? And can we avoid adding it to the `nativeMethods`?